ontology/kisao.md

```diff
@@ -24,6 +29,48 @@ license:
 activity_status: active
 repository: https://github.com/SED-ML/KiSAO
 preferredPrefix: KISAO
+usages:
+  - user: https://sed-ml.org/
+    type: annotation
+    description: The Simulation Experiment Description Markup Language (SED-ML) is a language for describing simulations and visualizations of their results.
+    examples:
+      - url: https://sed-ml.org/examples.html
+        description: "Several examples of simulations encoded in SED-ML"
+  - user: https://biosimulations.org/
+    type: annotation
+    description: BioSimulations is a repository of biosimulation projects.
+    examples:
+      - url: https://biosimulations.org/projects/Repressilator-Elowitz-Nature-2000
```

Comment:
How is KISAO used in this project? I cant see it form the page

Reply:
BioSimulators uses KiSAO to keep track of the type of each simulation. The name of the KiSAO term is displayed on the landing tab for each simulation.

For the example above (https://biosimulations.org/projects/Repressilator-Elowitz-Nature-2000), the simulation type is CVODE (KISAO_0000019).

Reply:
Thanks